src/System.IO.FileSystem/src/System/IO/MultiplexingWin32WinRTFileSystem.cs

```diff
@@ -145,33 +147,63 @@ private bool ShouldUseWinRT(string fullPath, bool isCreate)
 
             do
             {
-                Interop.WIN32_FIND_DATA findData = new Interop.WIN32_FIND_DATA();
-                using (SafeFindHandle handle = Interop.mincore.FindFirstFile(fullPath, ref findData))
-                {
-                    int error = Marshal.GetLastWin32Error();
+                int error = Interop.ERROR_SUCCESS;
```

Comment:
Can you add a comment to ShouldUseWinRT indicating what determines whether we should use WinRT or not? The method is sufficiently complicated that it's not clear on first-glance what the method is really looking for to make its decision.